internal/daysteps/daysteps.go

```diff
@@ -11,10 +18,48 @@ const (
 	mInKm = 1000
 )
 
+var ErrCritical = errors.New("critical error")
+var ErrConversion = errors.New("conversion error")
+
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	slice := strings.Split(data, ",")
+	if len(slice) != 2 {
+		return 0, time.Duration(0), fmt.Errorf("%w: Не переданы шаги или продолжительность прогулки", ErrCritical)
+	}
+
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return 0, time.Duration(0), fmt.Errorf("%w: %w", ErrConversion, err)
```

Comment:
Нельзя оборачивать две ошибки, для этого тогда нужно использовать `errors.Join`. По мне лучше суть ошибки писать на месте, чтобы проще читать код.
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```